### ISSUE TITLE
Ios ads suddenly stopped

### ISSUE THREAD

**Issue by liamkavfc** [marcojak/MTAdmob#73]
This isn’t a problem with the plug-in but I wonder if anyone has had this problem with AdMob too…

So when I put my app live my ads works perfect, 1 week later, the ads stopped appearing. I am getting request but match rate is 0%. All my account details are filled in etc. But now the test ads won’t even show in test mode now, anyone had this? The test ads still show in my test android app using the same Xaml element

**Comment by ShadowOfPhantom** [comment]
Hey! I faced the same and submitted the issue and solution here: https://github.com/marcojak/MTAdmob/issues/74
I don’t know if it’s applicable for iOS, I haven’t tested it yet because currently xcode 13 is unsupported but seems like it’s applicable for both android and ios. Check it out :)

**Comment by liamkavfc** [comment]
Yeah it's a weird one because my test ads started working randomly agin now but also my production ad got 1 impression yesterday randomly then went back down to 0 again so I'm not sure how it all works 😅

**Comment by ShadowOfPhantom** [comment]
> Yeah it's a weird one because my test ads started working randomly agin now but also my production ad got 1 impression yesterday randomly then went back down to 0 again so I'm not sure how it all works 😅

Just add your device to list of test devices in your code and use your production ad units as usual, it will work.  As far as i can say it’s the only way to show test ads, I still can’t use test ad units :)

**Comment by bijington** [comment]
I also appear to be seeing this behaviour. I don't know if this information helps at all but so far I have been able to identify these items that may be effecting it:

In the output window I see `<Google:HTML> You are currently using version 7.66.0 of the SDK. Please consider updating your SDK to the most recent SDK version (8.12.0) to get the latest features and bug fixes. The latest SDK can be downloaded from https://goo.gl/UoiJ8F. A full list of release notes is available at https://developers.google.com/admob/ios/rel-notes.` Given that the newest version is a major change there is the possibility that things could have changed enough to break things.

Currently the `Xamarin.Google.iOS.MobileAds` library is stuck on 7.66.0 and I can't see when a newer version could be supported yet. https://www.nuget.org/packages/Xamarin.Google.iOS.MobileAds/

In my `CrossMTAdmob.Current.OnRewardedVideoAdFailedToLoad` callback I am constantly receiving an `ErrorCode` of 8 which according to this site (https://developers.google.com/android/reference/com/google/android/gms/ads/AdRequest#ERROR_CODE_APP_ID_MISSING) the issue is due to `The ad request was not made due to a missing app ID.`. I think this backs up a potentially breaking change by Google.

I shall continue to investigate and report back my findings

**Comment by bijington** [comment]
Upon further investigation it does appear that the error code of 8 may not be referring to the App Id being missing... I see this message reported every time I receive a failed to load callback.

> Cannot find an ad network adapter with the name(s): com.google.DummyAdapter. Remember to link all required ad network adapters and SDKs, and set -ObjC in the 'Other Linker Flags' setting of your build target.

I haven't worked out the cause/fix for this yet though.

**Comment by bijington** [comment]
OK I finally have my ads loading again. There was no issue with this library or the others it depends on but I wanted to report how I solved mine.

Basically I had to add this line:
```
google.com, pub-3940256099942544, DIRECT, f08c47fec0942fa0
```
in to our `app-ads.txt` file. Now the documentation (https://developers.google.com/admob/ios/test-ads) suggests this is only needed for running test ads this has solved the issue.
